.circleci/rebase-pr.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * Rebases the current branch on top of the GitHub PR target branch.
+ *
+ * **Context:**
+ * Since a GitHub PR is not necessarily up to date with its target branch, it is useful to rebase
+ * prior to testing it on CI to ensure more up to date test results.
+ *
+ * **NOTE:**
+ * This script cannot use external dependencies or be compiled because it needs to run before the
+ * environment is setup.
+ * Use only features supported by the NodeJS versions used in the environment.
+ */
+// tslint:disable:no-console
+const {execSync: execSync_} = require('child_process');
```

Comment:
This renaming seems redundant. Why not just:

```suggestion
const {execSync} = require('child_process');
```